src/axolotl/prompt_tokenizers.py

```diff
@@ -77,7 +82,7 @@ def _tokenize(
             result = self.tokenizer(
                 prompt,
                 truncation=True,
-                max_length=self.sequence_len,
+                max_length=self.max_length,
```

Comment:
What is max_length for? Since you're splitting long texts in the prompter, I don't quite understand what this is for.

Reply:
the tokenizer no longer emits the entire tokenized string (it used to emit a warning), but now truncates it to max_length.

Reply:
ah I see. You split after tokenize, not before.